src/sdkV2.ts

```diff
@@ -408,6 +425,77 @@ export default class CegaEvmSDKV2 {
    * USER FACING METHODS
    */
 
+  // ====== lpCega Offramp (Vault Token Market) code starts here
+
+  async getOfframpFeeBps(): Promise<number> {
+    const cegaEntry = await this.loadCegaEntry();
+    return cegaEntry.getOfframpFeeBps();
+  }
+
+  async getOrderData(maker: EvmAddress, orderHash: string): Promise<GetOrderDataResponse> {
+    const cegaEntry = await this.loadCegaEntry();
+    return cegaEntry.getOrderData(maker, orderHash);
+  }
+
+  async hashOrder(order: LpCegaOfframpOrder): Promise<string> {
+    const cegaEntry = await this.loadCegaEntry();
+    return cegaEntry.hashOrder(order);
+  }
+
+  async signOrder(cegaEntry: ethers.Contract, order: LpCegaOfframpOrder): Promise<string> {
```

Comment:
why are you passing in cegaEntry?

why not just `loadCegaEntry()` like we do elsewhere?